exercises/sum.cpp

```diff
@@ -6,3 +6,30 @@
   Insert the second number: 2
   Sum: 3
 */
+
+
+
+#include <iostream> 
+using namespace std; 
+  
+
+int somma(int A, int B) 
+{ 
+ 
+    return A + B; 
+} 
+  
+
+int main() 
+{ 
+   
+    int primo;
+    int secondo; 
+     cout << "Inserisci primo: \n";
+      cin >> primo;
```

Comment:
Qui e nelle righe a seguire ci sono problemi di intendazione. Il testo deve essere allineato, in cin invede è qualche spazio più avanti rispetto al cout precedente